frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.tsx

```diff
@@ -133,6 +133,18 @@ export const ParametersWidgetContainer = styled(FullWidthContainer)<{
           ? "var(--mb-color-border)"
           : getDashboardBodyBgColor(isNightMode)};
     `}
+
+    ${({ isNightMode }) =>
+    isNightMode &&
+    css`
+      --mb-color-text-secondary: color-mix(
+        in srgb,
+        var(--mb-color-text-white) 65%,
+        transparent
+      );
+      --mb-color-border: var(--mb-color-text-medium);
```

Comment:
Wdyt if we try to use new colors from the updated palette right away? Probably `--mb-base-color-orion-60` or another Orion accent

Reply:
That'll do!

Reply:
Fixed in https://github.com/metabase/metabase/pull/49343/commits/d25658b9bff40d305aa95b5d51d97eeeaf2f25bd

This still results in the same colors on the fullscreen dashboard dark theme.
![Screenshot 2024-10-31 at 4 32 27 PM](https://github.com/user-attachments/assets/2cae4959-0c23-4f0b-9314-af5968dc22d0)